cardano-base/cardano-base.cabal

```diff
@@ -0,0 +1,36 @@
+cabal-version: 3.0
+name: cardano-base
+version: 0.0.0.0
+synopsis: Various utilities for Cardano
+description: Various utilities for Cardano.
+category:
+  Cardano,
+
+copyright: 2025 Input Output Global Inc (IOG).
+author: IOHK
+maintainer: operations@iohk.io
+license: Apache-2.0
+license-files:
+  LICENSE
+  NOTICE
+
+extra-doc-files:
+  CHANGELOG.md
+  README.md
+
+common project-config
+  default-language: GHC2021
```

Comment:
This deviates from all other packages in cardano-base
```suggestion
```
I would not be opposed to go in this direction in the future, but for now consistency is more important